html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get">
+    <div>
+      <label for="email">Email</label>
+    </br>
+      <input type="email" id="email" name="email" maxlength="12">
+    </div>
+    <div>
+      <label for="why-is-coding-cool?">Why is coding cool?</label>
+    </br>
+      <select name="options">
+        <option value="select an option" selected>Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+  </br>
+    <div>
+      <label for="Language">Favorite-Language(choose one)</label>
+    </br>
+    </br>
+      <input type="radio" name="language" value="HTML">html
```

Comment:
since there are three inputs here, you will need to have 3 labels. a common way to deal with radio inputs and labels is to wrap them like this:
```html
      <label>
        <input type="radio" name="favoriteLanguage" value="HTML">
        HTML
      </label>
```